synth.py

```diff
@@ -35,146 +35,27 @@
 
     s.move(library, excludes=excludes)
 
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    "from google.iam.v1 import policy_pb2 as",
-    "from google.iam.v1 import iam_policy_pb2_grpc as",
-)
-
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    "from google.iam.v1 import iam_policy_pb2_grpc "
-    "as google_dot_iam_dot_v1_dot_policy__pb2",
-    "from google.iam.v1 import iam_policy_pb2 "
-    "as google_dot_iam_dot_v1_dot_policy__pb2",
-)
-
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    "_ASSET\.fields_by_name\['iam_policy'\]\.message_type "
-    "= google_dot_iam_dot_v1_dot_policy__pb2\._POLICY",
-    "_ASSET.fields_by_name['iam_policy'].message_type = google_dot_iam_dot"
-    "_v1_dot_policy__pb2.google_dot_iam_dot_v1_dot_policy__pb2._POLICY",
-)
-
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    "_IAMPOLICYSEARCHRESULT\.fields_by_name\['policy'\]\.message_type "
-    "= google_dot_iam_dot_v1_dot_policy__pb2\._POLICY",
-    "_IAMPOLICYSEARCHRESULT.fields_by_name['policy'].message_type = google_dot_iam_dot"
-    "_v1_dot_policy__pb2.google_dot_iam_dot_v1_dot_policy__pb2._POLICY",
-)
-
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    "_IAMPOLICYANALYSISRESULT\.fields_by_name\['iam_binding'\]\.message_type "
-    "= google_dot_iam_dot_v1_dot_policy__pb2\._BINDING",
-    "_IAMPOLICYANALYSISRESULT.fields_by_name['iam_binding'].message_type = google_dot_iam_dot"
-    "_v1_dot_policy__pb2.google_dot_iam_dot_v1_dot_policy__pb2._BINDING",
-)
-
-_BORKED_ASSET_DOCSTRING = """\
-          The full name of the asset. For example: ``//compute.googleapi
-          s.com/projects/my_project_123/zones/zone1/instances/instance1`
-          `. See `Resource Names <https://cloud.google.com/apis/design/r
-          esource_names#full_resource_name>`__ for more information.
-"""
-
-_FIXED_ASSET_DOCSTRING = """
-          The full name of the asset. For example:
-          ``//compute.googleapis.com/projects/my_project_123/zones/zone1/instances/instance1``.
-          See https://cloud.google.com/apis/design/resource_names#full_resource_name
-          for more information.
-"""
-
-s.replace(
-    "google/cloud/asset_v*/proto/assets_pb2.py",
-    _BORKED_ASSET_DOCSTRING,
-    _FIXED_ASSET_DOCSTRING,
-)
-
-s.replace(
-    "google/cloud/**/asset_service_client.py",
-    "google-cloud-cloudasset",
-    "google-cloud-asset",
-)
-# Fix docstrings with no summary line
-s.replace(
-    "google/cloud/**/proto/*_pb2.py",
-    r''''__doc__': """Attributes:''',
-    ''''__doc__' : """
-    Attributes:''',
-)
-
-# Fix accesscontextmanager and orgpolicy imports
-s.replace(
-    "google/cloud/asset_v*/types.py",
-    "from google\.cloud\.asset_(v.*)\.proto import ((access_level_pb2)|(service_perimeter_pb2)|(access_policy_pb2))",
-    "from google.identity.accesscontextmanager.v1 import \g<2>",
-)
-
```

Comment:
Did you see any errors related to accesscontextmanager or orgpolicy? It's great if they "just work" now, I'm just surprised.

Reply:
I don't see any errors, they just worked :)